bnd.bnd

```diff
@@ -1,14 +1,14 @@
 Bundle-SymbolicName: com.ibm.ws.opentracing.zipkin
 Bundle-Name: Opentracing Zipkin
-Bundle-Description: Opentracing Zipkin Factory Implemention; version=2.0
+Bundle-Description: Opentracing Zipkin Factory Implemention; version=3.0
 Bundle-ManifestVersion: 2
-Bundle-Version: 2.0
+Bundle-Version: 3.0
 
 -dsannotations: com.ibm.ws.opentracing.zipkin.OpentracingZipkinTracerFactory 
 
-Import-Package: io.openliberty.opentracing.spi.tracer;version="[2.0,3)",io.opentracing;version="[0.33,1)", \
+Import-Package: io.openliberty.opentracing.spi.tracer;version="[3.0,4)",io.opentracing;version="[0.33,1)", \
  io.opentracing.propagation;version="[0.33,1)",io.opentracing.tag;version="0.33.0",javax.net.ssl,javax.net,org.osgi.service.component
```

Comment:
Please check if `javax` here is used or not.  If it isn't used, I prefer to remove them.